libultraship/libultraship/ConfigFile.cpp

```diff
@@ -58,6 +58,9 @@ namespace Ship {
 		(*this)["ARCHIVE"]["Main Archive"] = "oot.otr";
 		(*this)["ARCHIVE"]["Patches Directory"] = "";
 
+		(*this)["SAVE"]["Save Filename"] = "oot_save.sav";
```

Comment:
See how we handle the main OTR file setting. It takes a full relative path rather than splitting up the directory and filename.

Reply:
Will be resolved in the next commit.